### ISSUE TITLE
'"SoundCaster" is damaged and can't be opened'

### ISSUE THREAD

**Issue by hugovk** [dracos/soundcaster#2]
After downloading https://github.com/dracos/soundcaster/releases/download/v2.0/SoundCaster-node-bundled.zip, unzipping, and opening SoundCaster, I get this:

![image](https://cloud.githubusercontent.com/assets/1324225/24059779/f8ddb9f0-0b58-11e7-95c5-89286cd7ab38.png)

macOS Sierra 10.12.3, node 7.5.0.

Same thing with https://github.com/dracos/soundcaster/releases/download/v2.0/SoundCaster-node-bundled.zip

**Comment by dracos** [comment]
Sorry to hear that. I am also on Sierra 10.12.3. I get the same thing with the download, but my local installation is running fine. Console gives some code signing issue, so that'll be a delightful world of pain trying to remember what I did and what I've done wrong. Hohum...

**Comment by Roxxor91** [comment]
I was able to install it after "sudo spctl --master-disable" and it started.
Then I did "sudo spctl --master-enable" and it still started. So thats a workaround to get it installed.
Sadly it didn't show my Sony HT-XT2 which can be either function as DLNA or chromecast device, so I was not able to stream anything to it.(I preferably stream to the Chromecast part of it on Linux or from Android)

EDIT: Should I open an issue for my soundbar not being found?
